pkg/okteto/namespace.go

```diff
@@ -133,3 +133,21 @@ func validateNamespace(namespace, object string) error {
 	}
 	return nil
 }
+
+// SleepNamespace sleeps a namespace
+func (c *OktetoClient) SleepNamespace(ctx context.Context, namespace string) error {
```

Comment:
I would try to avoid as much as possible to include functions that are only for tests in the "production" code.